services/user.py

```diff
@@ -0,0 +1,54 @@
+from db.models import User
+from django.core.exceptions import ValidationError
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> list:
+    try:
+        user = User.objects.create_user(
+            username=username,
+            password=password,
+            email=email,
+            first_name=first_name or "",
+            last_name=last_name or ""
+        )
+        return user
+    except Exception as e:
+        raise ValidationError(
+            f"An error occurred while creating the user: {str(e)}"
+        )
+
+
+def get_user(user_id: int) -> User:
+    user = User.objects.get(pk=user_id)
+    return user
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> list:
+    if user_id:
+        user = User.objects.get(pk=user_id)
+        if username is not None:
```

Comment:
You could simplify check 
```suggestion
        if username:
```